server/static/js/admin/controllers.js

```diff
@@ -533,6 +533,29 @@ app.controller("SubmissionListCtrl", ['$scope', '$stateParams', '$window', 'Sear
     $scope.search = function() {
       $scope.getPage($scope.currentPage)
     }
+    
+    $scope.download_zip = function(query, all, courseId) {
+      filename = 'query_(your email)_(current time).zip'
+      Search.download_zip({
+        query: query,
+        all: all,
+        courseId: courseId
+      }, function() {
+        $window.swal({
+          title: 'Success',
+          text:'Saving submissions to ' + filename +
+          '\n Zip of submissions will be ready in Google Cloud Storage ok_grades_bucket in a few minutes',
+          type: 'success',
+          confirmButtonText: 'View zip',
+          cancelButtonText: 'Not now',
+          showCancelButton: true},
+          function() {
+            $window.location = 'https://console.developers.google.com/storage/browser/ok_grades_bucket/';
```

Comment:
I can add it in; it's actually just the filename appended to the URL in the JS.

Looks like Luise is right. For that matter though, only project owners, editors, and team members can access the exports atm.